app/models/manageiq/providers/cloud_manager/template.rb

```diff
@@ -122,6 +122,13 @@ def self.display_name(number = 1)
     n_('Image', 'Images', number)
   end
 
+  def self.tenant_id_clause(user_or_group)
+    template_tenant_ids = MiqTemplate.accessible_tenant_ids(user_or_group, Rbac.accessible_tenant_ids_strategy(MiqTemplate))
```

Comment:
this can be just

`accessible_tenant_ids(user_or_group, Rbac.accessible_tenant_ids_strategy(self))`